gateway-topology-hadoop-xml/src/main/java/org/apache/knox/gateway/topology/hadoop/xml/HadoopXmlResourceParser.java

```diff
@@ -70,6 +71,7 @@ public class HadoopXmlResourceParser implements AdvancedServiceDiscoveryConfigCh
   private static final String CONFIG_NAME_APPLICATION_PREFIX = "app";
   private static final String CONFIG_NAME_SERVICE_URL = "url";
   private static final String CONFIG_NAME_SERVICE_VERSION = "version";
+  public static final String REMOVE = "remove";
```

Comment:
We already have a constant for `remove` (see `CONFIG_NAME_PROVIDER_CONFIGS_PARAM_REMOVE` above). We may remove that one and use this new, more generic, contact name